packages/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -559,9 +559,9 @@ public void setShouldDeleteEnter(final ReactAztecText view, boolean shouldDelete
     }
 
     @Override
-    public void receiveCommand(final ReactAztecText parent, int commandType, @Nullable ReadableArray args) {
+    public void receiveCommand(final ReactAztecText parent, String commandType, @Nullable ReadableArray args) {
```

Comment:
Looks like not using the `receiveCommand(..., int ..., ...)` variant anymore ([which is deprecated alright](receiveCommand)) would let us also remove [the getCommandsMap override](https://github.com/WordPress/gutenberg/blob/436975285c0002e08cc33b62662e7549eff1e574/packages/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java#L554). No need to do it in this PR, just mentioning it for our future selves. We should open a ticket about it after merging the RN 0.64 upgrade.

Reply:
Sounds good 👍 I have limited knowledge around here so I only applied the minimum changes necessary to make this work, but I agree that this file might need some further refactoring.

Reply:
Created an issue for that: https://github.com/wordpress-mobile/gutenberg-mobile/issues/3635